server/controllers/entries.js

```diff
@@ -37,21 +38,21 @@ class Entry {
  *
  * @memberof Entry
 */
-  static getOne(request, response) {
-    for (let i = 0; i < entry.length; i++) {
-      if (entry[i].entryId === parseInt(request.params.entryId, 10)) {
-        return response.send({
-          message: 'Successful',
-          entry: data[i],
-          error: false
-        });
-      }
+static getOne(request, response) {
+  let index = entry.findIndex(item => item.entryId === parseInt(request.params.entryId, 10)); {
```

Comment:
Good job (y) Though if the value of index is not going to change, use `const` rather than `let`

Reply:
ok, will do.